### PR TITLE
GCC_2021-4123 [Task][Manager] Send mail FF14 to user's mail address when manager send a message to user

### PR DESCRIPTION
**Pull request by manhdat22**

### CODE REVIEW

**Comment by manhdat22** [comment]
Test

**Comment by manhdat22** [comment]
c

**Comment by manhdat22** [comment]
ccc

**Comment by manhdat22** [comment]
cc

**Comment by manhdat22** [comment]
casc

**Comment by manhdat22** [comment]
cascc

**Comment by manhdat22** [comment]
test

**Comment by manhdat22** [comment]
test thread

**Comment by manhdat22** [comment]
casc

**Comment by manhdat22** [comment]
c

**Comment by manhdat22** [comment]
asc

**Comment by manhdat22** [comment]
Tesr

**Comment by manhdat22** [comment]
asc

**Comment by manhdat22** [comment]
aa

**Comment by manhdat22** [comment]
okia

**Comment by manhdat22** [comment]
cc

**Review by datlm-1340 — APPROVED** [review]

**Comment by datlm-1340** [comment]
ccoke

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -16,5 +16,3 @@ def current_token
     request.env["warden-jwt_auth.token"]
   end
 end
```
Comment (datlm-1340) [inline app/controllers/sessions_controller.rb:18]: cccc

Comment (datlm-1340) [reply app/controllers/sessions_controller.rb:18]: c

Comment (datlm-1340) [reply app/controllers/sessions_controller.rb:18]: You cannot make a AJAX call to a local resource as the request is made using HTTP.

A workaround is to run a local webserver, serve up the file and make the AJAX call to localhost.

In terms of helping you write code to read JSON, you should read the documentation for `jQuery.getJSON()`

Comment (datlm-1340) [reply app/controllers/sessions_controller.rb:18]:
```ruby
request.env["warden-jwt_auth.token"]
```
You cannot make a AJAX call to a local resource as the request is made using HTTP.
A workaround is to run a local webserver, serve up the file and make the AJAX call to localhost.
In terms of helping you write code to read JSON, you should read the documentation for jQuery.getJSON()

**Comment by datlm-1340** [comment]
nah

**Review by datlm-1340 — APPROVED** [review]

**Comment by datlm-1340** [comment]
oke

**Review by datlm-1340 — APPROVED** [review]

**Review by datlm-1340 — APPROVED** [review]

**Review by datlm-1340 — APPROVED** [review]

**Review by datlm-1340 — APPROVED** [review]

**Comment by datlm-1340** [comment]
@manhdat22  e

**Comment by datlm-1340** [comment]
@manhdat22 yo

**Comment by datlm-1340** [comment]
Anh yêu em

**Comment by datlm-1340** [comment]
test

**Comment by datlm-1340** [comment]
okes

**Comment by datlm-1340** [comment]
ccccccq

**Comment by datlm-1340** [comment]
awt

**Comment by datlm-1340** [comment]
ccc

**Comment by datlm-1340** [comment]
cacasc

**Comment by datlm-1340** [comment]
cc

File: app/controllers/sessions_controller.rb (hunk 1 above)
Comment (datlm-1340) [reply app/controllers/sessions_controller.rb:19]: `oke`

Comment (datlm-1340) [reply app/controllers/sessions_controller.rb:19]: nah

Comment (datlm-1340) [reply app/controllers/sessions_controller.rb:19]: c

**Review by datlm-1340 — APPROVED** [review]

**Review by datlm-1340 — APPROVED** [review]

**Pull request merged**